data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_feats.lst

```diff
@@ -68,3 +68,10 @@ Shadow Walker VII			CATEGORY:FEAT	TYPE:ShadowWalker			VISIBLE:NO						PREABILITY
 Shadow Walker VIII		CATEGORY:FEAT	TYPE:ShadowWalker			VISIBLE:NO						PREABILITY:1,CATEGORY=FEAT,Shadow Walker VII																																																																																																		SOURCEPAGE:p.71
 Shadow Walker IX			CATEGORY:FEAT	TYPE:ShadowWalker			VISIBLE:NO						PREABILITY:1,CATEGORY=FEAT,Shadow Walker VIII																																																																																																		SOURCEPAGE:p.71
 Shadow Walker X			CATEGORY:FEAT	TYPE:ShadowWalker			VISIBLE:NO						PREABILITY:1,CATEGORY=FEAT,Shadow Walker IX																																																																																																		SOURCEPAGE:p.71
+
+
+##
+# These "feats" exist only for use for manipulation of other feats (such Magical Artisan) or to meet other prerequirements (such as the for prestige classes)
+#
+Craft Nar Demonbinder Sign		CATEGORY:FEAT		TYPE:ItemCreation	VISIBLE:NO		PRECLASS:1,Nar Demonbinder=2			DESC:Nar Demonbinders learn to create devices that act as calling diagrams or demon traps. A 2nd level Nar Demonbinder can craft an Iron Sign in three days costing 1000gp. A 4th level Nar Demonbinder can craft a Brazen Sign in five days costing 2000gp. A 6th level Nar Demonbinder can craft a Silver Sign in seven days for 5000gp.
+Craft Raumathari Sword Focus		CATEGORY:FEAT		TYPE:ItemCreation	VISIBLE:NO		PRECLASS:1,Raumathari Battlemage=1		DESC:A Raumathari Battlemage can, through a special ritual, modify a normal or magical weapon so that it becomes a spell focus increasing the battlemage's caster level and replacing any spell's material component with less than a 50gp value. This rite requires three days, requires a masterwork weapon, and has a material cost of 1000gp.
```

Comment:
Is this meant to be a valid feat or just a special access? VISIBLE:NO means it is not displayed anywhere. VISIBLE:EXPORT will at least display on the character sheet. VISIBLE:YES will give access to it as a real feat.

The PRECLASS is not necessary if VISIBLE is NO or EXPORT, unless there is a CHOOSER that will grant the feat if qualified.